### PR TITLE
Add Laravel 7 support

### LINKED ISSUE #39: Laravel 7 Support

### PR DESCRIPTION
**Pull request by kapersoft**
Fixes https://github.com/laravel-notification-channels/pushover/issues/39

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     "require": {
         "php": ">=7.2",
         "guzzlehttp/guzzle": "^6.2",
-        "illuminate/notifications": "5.5.*|5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/support": "5.5.*|5.6.*|5.7.*|5.8.*|^6.0"
+        "illuminate/notifications": "5.5.*|5.6.*|5.7.*|5.8.*|^6.0|^7.0",
```
Comment (atymic) [inline composer.json]:
```suggestion
        "illuminate/notifications": "~5.5 || ^6.0 || ^7.0",
```

Single pipe is deprecated + can clean up reqs here :)

Comment (atymic) [reply composer.json]: (and below)

Comment (atymic) [reply composer.json]: (also need to allow testbench 4/5 for laravel 6/7

**Review by atymic — CHANGES_REQUESTED** [review]

**Comment by kapersoft** [comment]
@atymic PR is updated with the requested changes

**Review by atymic — APPROVED** [review]

**Comment by casperboone** [comment]
Thx!

**Pull request merged**